tests/python/frontend/onnx/test_forward.py

```diff
@@ -4640,6 +4640,48 @@ def repeat(N, D):
     )
 
 
+def test_LinearRegressor():
```

Comment:
We probably should investigate why adding `@tvm.testing.uses_gpu` causes CI issues. `test_qlinearconv` might need this annotation too.

Reply:
add for LinearRregressor test, test_qlinearconv is known to fail for cuda